src/content/docs/en/guides/astro-db.mdx

```diff
@@ -654,6 +655,49 @@ When you're ready to deploy, see our [Deploy with a Studio Connection guide](#de
 
 </ReadMore>
 
+## LibSQL
+
+<p><Since pkg="@astrojs/db" v="0.14.0" /></p>
+
+Alternatively to Astro Studio, Astro DB can connect to any LibSQL server. You can use an offering from any platform that exposes the Hrana protocol of the server or host it yourself.
+
+To connecto Astro DB to a LibSQL instead of Studio, set the following environment variables:
+- `ASTRO_DB_REMOTE_URL`: the connection URL to your LibSQL server
+- `ASTRO_DB_APP_TOKEN`: the auth token to your LibSQL server
+
+The environment variables have to be set during `astro build`, `astro db push --remote` and during runtime if your project use Astro DB on SSR pages.
+
+Details of the LibSQL connection can be configured using the connection URL.
+
+### URL scheme and host
+
+LibSQL supports both HTTP and WebSockets as the transport protocol for for a remote server. It also supports using a local file or an in-memory DB. Those can be configured using the following URL schemes in the connection URL:
+
+- `memory:` will use an in-memory DB. Host must be empty in this case.
+- `file:` will use a local file. The host is the path to the file (`file:path/to/file.db`).
+- `libsql:` will use a remote server through the protocol preferred by the library (this might be different across versions). The host is the address of the server (`libsql://your.server.io`).
+- `libsql+http:` will use a remote server through HTTP. `libsql+https:` can be used to enable a secure connection. The host is the same as for `libsql:`.
+- `ws:` will use a remote server through WebSockets. `wss` can be used to enable a secure connection. The host is the same as for `libsql:`.
+
+### `encryptionKey`
+
+LibSQL has native support for encrypted databases. Passing this search parameter will enable encryption using the given key:
+
+- `file:path/to/file.db?encryptionKey=your-encryption-key`
+- `libsql://your.server.io?encryptionKey=your-encryption-key`
+
+### `syncUrl`
+
+Passing a separate connection URL on this search parameter to turn the DB into an embedded replica of another of another database. This should only be used with the schemes `file:` and `memory:`. The parameter must be URL encoded.
+
+Embedded replicas are a feature of LibSQL clients that enables a full synchronized copy of your database on a local file or in memory for ultra-fast reads. Writes are sent to a remote database defined on the `syncUrl` and synchronized with the local copy.
```

Comment:
```suggestion
Embedded replicas are a feature of LibSQL clients that enables a full synchronized copy of your database on a local file or in memory for ultra-fast reads. Writes are sent to a remote database defined on the `syncUrl` and synchronized with the local copy.

Use this property to pass a separate connection URL to turn the DB into an embedded replica of another of another database. This should only be used with the schemes `file:` and `memory:`. The parameter must be URL encoded.
```
These read better when the order is swapped!

Reply:
I noticed a repeat that seems wrong to me: `replica of another of another database` > `replica of another database` (first paragraph in the original, or second paragraph in Sarah's suggestion).